suite-native/atoms/src/Chip.tsx

```diff
@@ -13,10 +14,12 @@ type ChipProps = {
     icon: ReactNode;
     isSelected?: boolean;
     style?: NativeStyleObject;
+    titleColor?: CSSColor;
```

Comment:
CSSColor should be used nearly never, use theme colors

Reply:
This needs to be CSS color because that's the type of theme colors (viz screenshot). Altough that's theme color so it should be fine.
https://tppr.me/HCoRq

Reply:
Please use theme colors, CSSColor is meant as last resort usage for some edge cases which is not. You can easily use theme color in styles just do `utils.colors[titleColor]` and it's done